Tests/TartTests/DiskImageFormatTests.swift

```diff
@@ -0,0 +1,77 @@
+import XCTest
+@testable import tart
+
+final class DiskImageFormatTests: XCTestCase {
+  func testRawFormatIsAlwaysSupported() throws {
+    XCTAssertTrue(DiskImageFormat.raw.isSupported)
+    XCTAssertTrue(DiskImageFormat.raw.canCreate)
+  }
+
+  func testASIFFormatSupport() throws {
+    // ASIF should be supported on macOS 15+
+    if #available(macOS 15, *) {
+      XCTAssertTrue(DiskImageFormat.asif.isSupported)
+    } else {
+      XCTAssertFalse(DiskImageFormat.asif.isSupported)
+    }
+  }
+
+  func testFormatFromString() throws {
+    XCTAssertEqual(DiskImageFormat(rawValue: "raw"), .raw)
+    XCTAssertEqual(DiskImageFormat(rawValue: "asif"), .asif)
+    XCTAssertNil(DiskImageFormat(rawValue: "invalid"))
+  }
+
+  func testFormatArgument() throws {
```

Comment:
1. This test should probably be named `testCaseInsensitivity()` or something like that, `testFormatArgument()` is too vague.

2. There's no need to have `"raw"` and `"invalid"` test cases here as they're already tested in `testFormatFromString ()`.